.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,243 @@
+# Contributing
+
+👍🎉 If you’re here to contribute to this project: thanks for taking the time! 🎉👍
+
+The following is a set of guidelines for contributing to this project.
+The goal is to make things go more smoothly. If you think that's not the case, feel free to propose changes.
+
+
+## Bug reports
+
+If you find a bug or a documentation issue, please [report](https://github.com/dexidp/helm-charts/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.md) it or even better: fix it.
+
+Read on to learn more about submitting changes.
+
+
+## Feature requests
+
+If you think a feature is missing, please [report](https://github.com/dexidp/helm-charts/issues/new?assignees=&labels=kind%2Fenhancement&template=feature_request.md) it or even better: implement it.
+
+If you report it, please describe your use case concretely and with as much details as possible.
+Hiding details behind abstractions makes it harder to understand the problem.
+
+If you decide to take a stab at implementing the feature,
+read the rest of this guide to learn more about submitting changes.
+
+
+## Opening a pull request
+
+The project accepts changes in the form of pull requests (or PRs).
+Pull requests allow maintainers to properly review the patch before accepting it
+and lets the CI ([GitHub Actions](https://docs.github.com/en/actions)) run automated checks to make sure the proposed change doesn't break anything.
+
+If you are new to GitHub or the concept of pull requests, check out the [official GitHub documentation](https://help.github.com/articles/about-pull-requests/).
+
+
+### Prerequisites
+
+First of all, you are going to need [Helm](https://helm.sh/docs/intro/install/) in order to test your changes.
+You can find the currently used version in the [.tool-versions](https://github.com/dexidp/helm-charts/blob/master/.tool-versions) file at the root of this repository.
+
+Alternatively, you can use [asdf](https://asdf-vm.com/) to install the right version.
+
+You will also need a Kubernetes cluster to test your changes.
+One option is using [kind](https://kind.sigs.k8s.io/) and installing the updated chart in a local cluster.
+An alternative is using a managed Kubernetes provider, like [DigitalOcean](https://www.digitalocean.com/).
```

Comment:
Is there any specifics around how the clusters need to be set up? Or are the getting started instructions for both of these options sufficient to get set up for this process?

Reply:
> Is there any specifics around how the clusters need to be set up?

Not really. It might be worth mentioning that the chart either needs an ingress controller to expose the service publicly, or it can also be tested using `kubectl port-forward`.

I'm not sure how much detail we should be going into.

In order to test the chart the contributor first needs to understand how it works, including the potential requirement for an ingress controller.